hudi-common/src/main/java/org/apache/hudi/common/table/log/BaseHoodieLogRecordReader.java

```diff
@@ -136,16 +136,14 @@ public abstract class BaseHoodieLogRecordReader<T> {
   // Use scanV2 method.
   private final boolean enableOptimizedLogBlocksScan;
   protected FileGroupRecordBuffer<T> recordBuffer;
+  // Allows to consider inflight instants while merging log records
```

Comment:
Can we add more doc to address that this is only used for MDT RLI index, and give an use case how it works.

Reply:
Addressed